packages/back-end/src/controllers/config.ts

```diff
@@ -157,7 +152,7 @@ export async function getExperimentsScript(
       }
 
       if (exp.status === "stopped") {
-        if (exp.results === "won") {
+        if (exp.results === "won" || exp.results === "lost") {
```

Comment:
Hi @Auz ,

Don't you think that if the experiment have lost, `data.force` should equal `0` instead of `exp.winner` ?

```typescript
if (exp.status === "stopped") {
    if (exp.results === "won") {
        data.force = exp.winner;
    } else if (exp.results === "lost") {
        data.force = 0;
    } else {
       return;
    }
}
```

Best,
Manu

Reply:
winner is set to 0 in the case of a loss (which isn't great naming). My concern was about making the assumption that 0 was always the control.

Reply:
Make sense. Thanks for the details.

Reply:
Thinking more about it we agree with you. 0 (control) should be returned for all other experiment results.